packages/plugin-exec/sources/execUtils.ts

```diff
@@ -43,7 +43,7 @@ export async function loadGeneratorFile(range: string, protocol: string, opts: F
 
   // If the file target is an absolute path we can directly access it via its
   // location on the disk. Otherwise we must go through the package fs.
-  const parentFetch = ppath.isAbsolute(path)
+  const parentFetch: FetchResult = ppath.isAbsolute(path)
```

Comment:
Why do you need to make these explicit?

Reply:
Because otherwise TypeScript yells at me:
`Property 'releaseFs' does not exist on type 'FetchResult | { packageFs: CwdFS; prefixPath: PortablePath; localPath: PortablePath; }'.
  Property 'releaseFs' does not exist on type '{ packageFs: CwdFS; prefixPath: PortablePath; localPath: PortablePath; }'.`